### ISSUE TITLE
MercadoPago/Paypal Integration

### ISSUE THREAD

**Issue by strteamservidores** [msmaiaa/csgo-brpanel#25]
Seria possível uma integração de pagamentos com o Paypal e MercadoPago?

**Comment by msmaiaa** [comment]
@strteamservidores sim, já existia com o paypal porém eu removi. Não olhei a integração com o mercado pago porém creio que seja simples de integrar.
